e2e/content-services/comments/comment_component.e2e.ts

```diff
@@ -169,4 +174,52 @@ describe('Comment Component', () => {
         expect(commentsPage.getUserName(0)).toEqual(userFullName);
         expect(commentsPage.getTime(0)).toMatch(/(ago|few)/);
     });
+
+    describe('Consumer Permissions', () => {
+        let site, pngUploadedFile;
+
+        beforeAll(async (done) => {
+            await this.alfrescoJsApi.login(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+
+            site = await this.alfrescoJsApi.core.sitesApi.createSite({
+                title: Util.generateRandomString(8),
+                visibility: 'PUBLIC'
+            });
+
+            await this.alfrescoJsApi.core.sitesApi.addSiteMember(site.entry.id, {
+                id: acsUser.id,
+                role: CONSTANTS.CS_USER_ROLES.CONSUMER
+            });
+
+            pngUploadedFile = await uploadActions.uploadFile(this.alfrescoJsApi, pngFileModel.location, pngFileModel.name, site.entry.guid);
+
+            loginPage.loginToContentServicesUsingUserModel(acsUser);
+
+            contentServicesPage.navigateToDocumentList();
+
+            done();
+        });
+
+        afterAll(async (done) => {
+
+            await this.alfrescoJsApi.login(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
```

Comment:
you don't need to login aother time here and at this point from the afterAll can be removed the async
and the await inside